src/frontend/get-github-setup.ts

```diff
@@ -1,23 +1,22 @@
 import { jiraDomainOptions } from "./validations";
-import { NextFunction, Request, Response } from "express";
-import { getGitHubConfigurationUrl } from "../util/getUrl";
+import { Request, Response } from "express";
+import { getJiraMarketplaceUrl } from '../util/getUrl';
 
 /*
 When this request is made: Installing from Jira Marketplace - GitHub org does not have Jira installed.
 Redirects users back to github/configuration to install their Jira instance in GitHub org/s.
 If the installation was done from Jira Marketplace, the app is already installed.
 */
-export default (req: Request, res: Response, next: NextFunction): void => {
+export default (req: Request, res: Response): void => {
```

Comment:
Is it flow completely broken now? Should we feature-flag the change?

Reply:
from github installation, yeah, it's fairly broken.  Good thing that most customers don't install it from the github side first :)

Reply:
Nah, I don't think we should FF. I should've have FF this original change but, given that the install flow from GH now already broken, I think we just get this merged as is.